dotcom-rendering/src/lib/getBridgetVersion.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react';
+import { getEnvironmentClient } from './bridgetApi';
+
+const isCorrectBridgetVersion = (
+	version: string,
+	requiredVersion: number,
+): boolean => {
+	const majorVersion = parseInt(version);
+	return majorVersion >= requiredVersion;
+};
+
+export const useIsBridgetCompatible = (
+	requiredVersion = 2,
+): boolean | undefined => {
+	const [isCompatible, setIsCompatible] = useState<boolean | undefined>(
+		undefined,
+	);
+
+	useEffect(() => {
+		void getEnvironmentClient()
+			.nativeThriftPackageVersion()
+			.then((bridgetVersion) => {
+				setIsCompatible(
+					isCorrectBridgetVersion(bridgetVersion, requiredVersion),
+				);
+			})
```

Comment:
it might be worth indicating that we're only comparing major versions here, because right now I think  we have this situation:

- if `nativeThriftPackageVersion` resolves to `2.0.0`, calling `useIsBridgetCompatible(2.1)` returns `false`, but I think it _should_ return `true`

```suggestion
export const useIsBridgetCompatible = (
	requiredMajorVersion = 2,
): boolean | undefined => {
	const [isCompatible, setIsCompatible] = useState<boolean | undefined>(
		undefined,
	);

	useEffect(() => {
		void getEnvironmentClient()
			.nativeThriftPackageVersion()
			.then((bridgetVersion) => {
				setIsCompatible(
					isCorrectBridgetVersion(bridgetVersion, Math.floor(requiredMajorVersion)),
				);
			})
```